src/main/resources/swagger.api/records-editor.yaml

```diff
@@ -47,14 +47,6 @@ paths:
           schema:
             type: string
             format: uuid
-        - name: lang
-          in: query
-          required: false
-          description: Requested language. Optional. [lang=en]
-          schema:
-            type: string
-            pattern: '[a-zA-Z]{2}'
-            default: 'en'
```

Comment:
Shouldn't module descriptor be updated?

Reply:
It was optional, never used, and request will not fail in case it will be part of query. So, I think it shouldn't :)